CI failure on some architectures due to missing script

**Environment**
- libunwind version: master
- architecture: some
- OS: N/A

**Description**

The CI-unix builds are failing on some architectures because the run-check-namespace script was removed. It needs to be replaced with the check-namespace.sh script.
```
2024-06-03T18:46:24.7775663Z /home/runner/work/_temp/1527d875-08a0-460d-b055-f7f4b180c34b.sh: line 8: /home/runner/work/libunwind/libunwind/tests/run-check-namespace: No such file or directory
2024-06-03T18:46:24.7788807Z ##[error]Process completed with exit code 127.
```

--- a/.github/workflows/CI-unix.yml
+++ b/.github/workflows/CI-unix.yml
@@ -147,7 +147,7 @@ jobs:
             *)
                 srcdir=$(pwd)
                 cd build/tests
-                ${srcdir}/tests/run-check-namespace
+                ./check-namespace.sh
                 ;;
             esac
 

